src/utils/bytes.cairo

```diff
@@ -1,5 +1,11 @@
 from starkware.cairo.common.alloc import alloc
-from starkware.cairo.common.math import unsigned_div_rem, split_int, split_felt
+from starkware.cairo.common.math import (
+    unsigned_div_rem,
+    split_int,
+    split_felt,
```

Comment:
```suggestion
```

Reply:
Those are used in other functions from the files